gradle-lombok-plugin/src/main/groovy/io/franzbecker/gradle/lombok/LombokPlugin.groovy

```diff
@@ -25,6 +25,7 @@ class LombokPlugin implements Plugin<Project> {
 
     static final String NAME = "io.franzbecker.gradle-lombok"
     static final String LOMBOK_CONFIGURATION_NAME = COMPILE_ONLY_CONFIGURATION_NAME
+    static final String VERIFY_LOMBOK_CLASSPATH_CONFIGURATION_NAME = COMPILE_CLASSPATH_CONFIGURATION_NAME
```

Comment:
The point of using the compile only configuration was to not include the Lombok dependency in the compile class path – as it shouldn't be required at runtime. 🤔

Reply:
Yes, and this doesn't change that. It only changes which configuration the `verifyLombok` task uses to retrieve the Lombok JAR for verification.

Maybe I should also add that recent versions of Gradle have introduced to separate "compile classpath" concepts - there's the `compileOnly` configuration for the library's own compile classpath, and the `compileOnlyApi` configuration for the downstream compile classpath for library consumers. So that's why the `verifyLombok` task can retrieve the JAR from the "compile classpath" even though that same JAR won't be on the "compile classpath" for downstream projects.